bears/vcs/git/GitCommitBear.py

```diff
@@ -124,6 +132,9 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
                 yield Result(self, 'HEAD commit has no message.')
             return
 
+        yield from self.check_auth_email(
+            auth_email[0][:-1],  # Taking the email and removing \n
```

Comment:
Input sanitation should be done as early as possible, in this case, when you first get the email adress.